atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -44,12 +44,14 @@
 from dockerfile_parse import DockerfileParser
 from atomic_reactor.plugin import PreBuildPlugin
 import json
+import datetime
 
 
 class AddLabelsPlugin(PreBuildPlugin):
     key = "add_labels_in_dockerfile"
 
-    def __init__(self, tasker, workflow, labels, dont_overwrite=("Architecture", )):
+    def __init__(self, tasker, workflow, labels, dont_overwrite=("Architecture", ),
+                 auto_labels=()):
```

Comment:
It's generally good practice not to use mutable default arguments, but instead have them default to None and describe what they will be set to in the docstring.